.github/workflows/python-app.yml

```diff
@@ -0,0 +1,45 @@
+
+name: selective-disclosure-jwt-demo
+
+on:
+  push:
+    branches: '*'
+  pull_request:
+    branches: '*'
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version:
+          - '3.8'
+          - '3.9'
+          - '3.10'
```

Comment:
Considering this is more a demo/reference code, not a library, I don't think we need to run this on three python versions. But I'm fine either way.

Reply:
moving on the willing to migrate this PoC to a real python package and also considering the fact that an analyst may have different python versions this is a quite standard CI